packages/analytics-react-native/README.md

```diff
@@ -148,3 +148,13 @@ track('Button Clicked').promise.then((result) => {
   result.message; // "Event tracked successfully" (The response message)
 });
 ```
+
+### User Log out
+
+Reset the user (usually called after the user logs out in the app). This will clear the `userId` and user properties (previously set using `Identify`)
```

Comment:
`reset()` only updates user ID and device ID sent in event payload, but does not necessarily clear user properties set using Identify. The user property in identify calls remain in the user's identity to which the identify was called for. After calling `reset()` the succeeding events now belong to a new user identity.

Reply:
Done. Updated.
Take a look now.

Feel free to edit the file and add any change you want.